identity-android/src/main/java/com/android/identity/SessionEncryption.java

```diff
@@ -196,21 +264,38 @@ void setSendSessionEstablishment(boolean sendSessionEstablishment) {
      * @return the bytes of the <code>SessionEstablishment</code> or <code>SessionData</code>
      *     CBOR as described above.
      */
-    public @NonNull byte[] encryptMessageToDevice(@Nullable byte[] messagePlaintext,
-            @NonNull OptionalLong statusCode) {
-        return encryptMessageToDeviceHelper(messagePlaintext, statusCode, false);
+    public @NonNull byte[] encryptMessage(@Nullable byte[] messagePlaintext,
+                                          @NonNull OptionalLong statusCode) {
+        return encryptMessageHelper(messagePlaintext, statusCode, false);
     }
 
     // Only used for testing, will produce a SessionEstablishment message with an
     // invalid COSE_Key for EReaderKeyq
-    @NonNull byte[] encryptMessageToDeviceWithInvalidEReaderKey(
+    @NonNull byte[] encryptMessageWithInvalidEReaderKey(
             @Nullable byte[] messagePlaintext,
             @NonNull OptionalLong statusCode) {
-        return encryptMessageToDeviceHelper(messagePlaintext, statusCode, true);
+        return encryptMessageHelper(messagePlaintext, statusCode, true);
+    }
+
+    /**
+     * Create a SessionData message (as defined in ISO/IEC 18013-5 9.1.1.4 Procedure) with a status
+     * code and no data.
+     *
+     * @param statusCode the intended status code, with value as defined in ISO/IEC 18013-5 Table 20.
+     * @return a byte array with the encoded CBOR message
+     */
+    static public @NonNull byte[] encodeStatus(long statusCode) {
+        CborBuilder builder = new CborBuilder();
+        MapBuilder<CborBuilder> mapBuilder = builder.addMap();
+        mapBuilder.put("status", statusCode);
+        mapBuilder.end();
+        return Util.cborEncode(builder.build().get(0));
     }
 
     /**
-     * Decrypts a message received from the remote mDL prover.
+     * Decrypts a message received from a receiver of the opposite type (ie. {@link SessionEncryption}
+     * objects with the ROLE_MDOC role will encrypt messages with the remote mDL prover as the
+     * intended receiver.
```

Comment:
Same comment as for `encryptMessage()` and just say _Decrypts a message received from the remote device_.